### DISCUSSION TITLE
Issue when I run Make

### DISCUSSION THREAD

**Discussion by VOLKERTH** [cockpit-project/cockpit-machines#1008] [General]
Hi all,

I'm getting an issue when I run "make" inside the folder /cockpit-machines (like the instructions)

Please, can you help me?

Thanks!

![image](https://user-images.githubusercontent.com/58624086/230049133-8084f8bc-264a-4ab9-9bb2-f5d2318be4d7.png)

**Comment by KKoukiou** [comment]
Probably you 're using old nodejs. Can you show the output of `node --version`

**Reply by VOLKERTH** [reply]
![image](https://user-images.githubusercontent.com/58624086/230068644-8c2f09e6-48c7-4b36-8a30-d6b3db4d1d88.png)

**Reply by jelly** [reply]
Try updating to nodejs 16 or try our toolbox container for development https://github.com/cockpit-project/cockpit/blob/main/HACKING.md#setting-up-development-container

**Reply by VOLKERTH** [reply]
I have installed Nodejs 18 and it works!! Thanks a lot!

**Comment by VOLKERTH** [comment]
Now is running!
